examples/tutorials/03_ready_to_use_model_tf.ipynb

```diff
@@ -41,7 +41,40 @@
    "metadata": {},
```

Comment:
<p>Split <code>predict_model</code> to multiple lines</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>